src/inference/src/dev/core_impl.cpp

```diff
@@ -64,6 +64,13 @@ void stripDeviceName(std::string& device, const std::string& substr) {
     }
 }
 
+void validDeviceName(const std::string& device) {
```

Comment:
```suggestion
void validate_device_name(const std::string& device) {
```
New code should apply snake case code style.